test/io/RootImporter.test.cc

```diff
@@ -6,6 +6,7 @@
 //! \file RootImporter.test.cc
 //---------------------------------------------------------------------------//
 
+#include <algorithm>
 #include "io/RootImporter.hh"
```

Comment:
To catch potential missing headers (like <algorithm>, ha ha) in the class being tested, it's a best practice to put the corresponding class include first:

```suggestion
#include "io/RootImporter.hh"

#include <algorithm>
```